### ISSUE TITLE
INI settings freaks out on non-windows servers

### ISSUE THREAD

**Issue by gikeymarcia** [gikeymarcia/Collector#118]
@adamblake merged the ini-setting branch into dev and tried to upload it to cogfog then my own server and got errors in both places.

From my server:
````
Warning: file_get_contents(home/gikeymarcia/gikeymarcia.com/Ravens/Experiment/Config.ini): failed to open stream: No such file or directory in /home/gikeymarcia/gikeymarcia.com/Ravens/Code/Parse.php on line 104

Warning: require(home/gikeymarcia/gikeymarcia.com/Ravens/Code/customFunctions.php): failed to open stream: No such file or directory in /home/gikeymarcia/gikeymarcia.com/Ravens/Code/initiateCollector.php on line 25

Fatal error: require(): Failed opening required 'home/gikeymarcia/gikeymarcia.com/Ravens/Code/customFunctions.php' (include_path='.:/usr/local/lib/php:/usr/local/php5/lib/pear') in /home/gikeymarcia/gikeymarcia.com/Ravens/Code/initiateCollector.php on line 25
````

From cogfog:
````

Warning: file_get_contents(home/cogfogweb/cogfog.com/VY/SoW/Ravens/Experiment/Config.ini): failed to open stream: No such file or directory in /home/cogfogweb/cogfog.com/VY/SoW/Ravens/Code/Parse.php on line 104

Fatal error: Uncaught exception 'InvalidArgumentException' with message 'The given config file 'home/cogfogweb/cogfog.com/VY/SoW/Ravens/Experiment/Config.ini' is empty or of an unsupported file type (supported: YAML, JSON, INI.' in /home/cogfogweb/cogfog.com/VY/SoW/Ravens/Code/Parse.php:30 Stack trace: #0 /home/cogfogweb/cogfog.com/VY/SoW/Ravens/Code/initiateCollector.php(22): Parse::fromConfig('home/cogfogweb/...', true) #1 /home/cogfogweb/cogfog.com/VY/SoW/Ravens/index.php(19): require('/home/cogfogweb...') #2 {main} thrown in /home/cogfogweb/cogfog.com/VY/SoW/Ravens/Code/Parse.php on line 30
````

@TysonKerr , any chance you'd know how we can address this?

**Comment by TysonKerr** [comment]
So, this is caused by the `FileLocations::clean` function removing the trailing slash for the path `/home/cogfogweb/cogfog.com/...`. A simple enough fix. However, then the problem immediately crashes on login.php, because I still havent merged in my fix for `getTrialTypeFiles()` not working with case-sensitive absolute dirs. But, I almost have the solution completed, so I will upload this fix asap, and then this branch should be able to run without problems.

**Comment by TysonKerr** [comment]
fixed in 2ad53a94251ecdcfa8bcffbdc3096a65c2ca6577
